### ISSUE TITLE
[SPIKE] 508-defect-3: Instructions on how to use the benefit letter download accordions rely on visual symbols 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#45999]
## Point of contact

Josh Kim

## List of issues by severity
There are 3 tickets related to this issue: 
Main ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45650
Heading: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45944
Labelling: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45945
Responsive: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45946
Download: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45998
Visual Instructions: https://github.com/department-of-veterans-affairs/va.gov-team/issues/45999
I've grouped these issues together as they're closely related and dependent on one another. Whatever solution we consider will likely need to take into account all of these issues together from a holistic perspective @skylerschain 

### Instructions rely on visual symbols (Moderate)
The first paragraph instructs users to expand boxes with "+"- a visual symbol that is not announced to screen reader users in the accordions. An immediate bandaid fix may be to align the aural experience with the instructions e.g., "To see an explanation of each letter, expand the _buttons_ below." However, this is likely a workaround to make up for an suboptimal design pattern.

https://user-images.githubusercontent.com/14154792/184918282-e7d465e0-0081-4fda-baed-65d5eaefcd53.MP4

## Reproduction steps

1. [Log in as user 299 and get to step 2](https://staging.va.gov/records/download-va-letters/letters/letter-list)
2. Open the benefit summary accordion

## Proposed solution or next steps

TBD, this requires further exploration with design, engineering, and content. My immediate thoughts are:
- we should look into past research on why the designers choose the accordion and table pattern (for example, how complex can these tables be? e.g. for military service history); if no strong design reasons exist, we may want to consider redesigning this
- we should consider using plain checkboxes (instead of checkboxes within a table)
- depending on the length of this, it may deserve to live on a separate page 

## Open Questions

- Does this implementation of accordions differ from the way the design system is supposed to work? What's the accessible form of this component? 
- Is an accessible solution here one in which we just change the instructional text to not mention the + symbol? 

## Type of issue

- [x] Screenreader
- [x] Keyboard
- [ ] Focus
- [x] Headings
- [ ] Color
- [x] Zoom
- [x] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by jacobworrell** [comment]
Point as a 1 to answer open question.
